common/Cargo.toml

```diff
@@ -1,31 +1,29 @@
 [package]
 name = "shuttle-common"
-version = "0.7.3"
-edition = "2021"
-license = "Apache-2.0"
+version.workspace = true
+edition.workspace = true
+licence.workspace = true
 description = "Common library for the shuttle platform (https://www.shuttle.rs/)"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-anyhow = { version = "1.0.66", optional = true }
-async-trait = { version = "0.1.58", optional = true }
-axum = { version = "0.6.0", optional = true }
-chrono = { version = "0.4.23", features = ["serde"] }
+anyhow = { workspace = true, optional = true }
+async-trait = { workspace = true , optional = true }
+axum = { workspace = true, optional = true }
+chrono = { workspace = true, features = ["serde"] }
 comfy-table = { version = "6.1.3", optional = true }
 crossterm =  { version = "0.25.0", optional = true }
 http = { version = "0.2.8", optional = true }
-once_cell = "1.16.0"
+once_cell = { workspace = true }
 reqwest = { version = "0.11.13", optional = true }
 rustrict = "0.5.5"
-serde = { version = "1.0.148", features = ["derive"] }
-serde_json = { version = "1.0.89", optional = true }
+serde = { workspace = true, features = ["derive"] }
+serde_json = { workspace = true, optional = true }
 strum = { version = "0.24.1", features = ["derive"] }
-tracing = "0.1.37"
-uuid = { version = "1.2.2", features = ["v4", "serde"] }
+tracing = { workspace = true }
+uuid = { workspace = true, features = ["v4", "serde"] }
 
 [features]
-default = ["models"]
```

Comment:
Dependencies inherited from the workspace cannot set the default-features flag, so I figured we may as well remove this. For reference: https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#inheriting-a-dependency-from-a-workspace